workspace-manager-clienttests/src/main/java/scripts/utils/CloudContextMaker.java

```diff
@@ -0,0 +1,50 @@
+package scripts.utils;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+
+import bio.terra.workspace.api.WorkspaceApi;
+import bio.terra.workspace.model.CloudPlatform;
+import bio.terra.workspace.model.CreateCloudContextRequest;
+import bio.terra.workspace.model.CreateCloudContextResult;
+import bio.terra.workspace.model.JobControl;
+import bio.terra.workspace.model.JobReport;
+import java.util.UUID;
+import java.util.concurrent.TimeUnit;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class CloudContextMaker {
+  private static final Logger logger = LoggerFactory.getLogger(CloudContextMaker.class);
+  private static final long CREATE_CONTEXT_POLL_SECONDS = 10;
```

Comment:
nit: even when only used in sleep, prefer `Duration` to primitives for time durations.

Reply:
makes sense, this is really storing both a value and unit.